//package core.io.protocol;
message DseMainScreenReady
{
	
}